### ISSUE TITLE
Add visitor to associate PROGRAM references back to copybooks.

### ISSUE THREAD

**Issue by traceyyoshima** [moderneinc/rewrite-cobol#62]
TLDR:
A part impact analysis of `COBOL` changes through Oraweb searches relies on identifying program references in `CopyBooks`. However, `CopyBooks` and `COBOL` are essentially plaintext files until AFTER two preprocesses have been executed: `Copy` and `Replace`.

After the LST is produced we can visit the `COBOL` code, find words with `CopyStatement`s and associate the results back to the `CobolPreprocessor` LST through the `CopyBook`. I.E. link the context derived from the resolved `COBOL` back to the preprocessed code.

Requirements to limit the visitor to applicable CopyBooks:
https://github.com/moderneinc/rewrite-cobol/issues/61

**Comment by traceyyoshima** [comment]
Fixed, copybooks are visitable
